source/Coop.Core/Server/Connections/States/ResolveCharacterState.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Common.Messaging;
+
+namespace Coop.Core.Server.Connections.States
+{
+    public class ResolveCharacterState : ConnectionStateBase
+    {
+        public ResolveCharacterState(IConnectionLogic connectionLogic, IMessageBroker messageBroker) 
```

Comment:
This class is not completable until #315 is complete. Once the new states are implemented it can be approved.